src/scenes/AcceleratorRouter/ParticipantProjects/EditView/index.tsx

```diff
@@ -263,6 +270,40 @@ const EditView = () => {
               userLabel={strings.BY}
             />
           </Grid>
+          <Grid container>
+            <Grid item xs={12} margin={`0 ${theme.spacing(2)}`}>
+              <Typography fontSize='20px' fontWeight={600} lineHeight='28px'>
+                {strings.CARBON}
+              </Typography>
+            </Grid>
+            <MinMaxCarbonTextfield
+              label={strings.MIN_MAX_CARBON_ACCUMULATION}
+              onChange={onChangeParticipantProject}
+              valueMax={participantProjectRecord?.maxCarbonAccumulation}
+              valueMin={participantProjectRecord?.minCarbonAccumulation}
+            />
+            <ProjectFieldTextfield
+              id={'carbonCapacity'}
+              label={strings.CARBON_CAPACITY_TC02_HA}
+              onChange={onChangeParticipantProject}
+              type={'number'}
+              value='770'
```

Comment:
Should these be empty instead?

Reply:
Values are available from the API now, will update